code/game/area/areas.dm

```diff
@@ -282,6 +284,9 @@ GLOBAL_LIST_EMPTY(teleportlocs)
 		//Lockdown airlocks
 		for(var/obj/machinery/door/DOOR in RA)
```

Comment:
This should just be typecasted as airlock. Unless I'm wrong and regular doors can be locked...